components/tag/Tag.razor.cs

```diff
@@ -78,23 +91,48 @@ private static bool IsPresetColor(string color)
             }
 
             bool result = Regex.IsMatch(color, "^(pink|red|yellow|orange|cyan|green|blue|purple|geekblue|magenta|volcano|gold|lime)(-inverse)?$");
-            if (!result) result = Regex.IsMatch(color, "^#([0-9a-fA-F]{6}|[0-9a-fA-F]{3})$");
             return result;
         }
 
+        private static bool IsCustomColor(string color)
+        {
+            if (string.IsNullOrEmpty(color))
+            {
+                return false;
+            }
+
+            return color.StartsWith("#");
```

Comment:
It should not only support `#`, it should also support other formats, such as RGB or native color names.